client/app/pages/queries/query.html

```diff
@@ -71,10 +71,9 @@ <h3>
 
 </div>
 
-<div class="query-fullscreen" ng-app="angularResizable">
```

Comment:
@kocsmy FYI, this was needed. I assume you found this in some code sample? I wonder if the `<main>` and `<nav>` tags are there for similar reasons?